liquid_node/configuration.py

```diff
@@ -201,6 +216,24 @@ def __init__(self):
 
         self.timestamp = int(time.time())
 
+        self.liquid_apps = {}
+        for app in self.ALL_APPS:
+            self.liquid_apps[app] = {
+                'title': self.APP_TITLE.get(app) or app.title(),
+                'url': self.app_url(app),
+                'enabled': self.is_app_enabled(app),
+                'description': self.APP_DESCRIPTION[app],
+                'adminOnly': False,
+            }
+
+        self.liquid_apps['nextcloud-admin'] = {
+            'title': "Nextcloud",
```

Comment:
Put the title as "Nextcloud Admin" and the description below as "will log you in as the Nextcloud admin user. You may need to log out of Nextcloud first."